src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -138,12 +138,12 @@ def test_0002_bz2018432(multihost, newhostname, adjoin):
         domain_list_cmd = multihost.client[0].run_command(
             'sssctl domain-list', raiseonerr=False)
         domain_list = domain_list_cmd.stdout_text.split('\n')
-        domain_list.remove("implicit_files")
```

Comment:
If we make sure that implicit files is not present and prune it if it is, then the test stays backward compatible.
```
if "implicit_files" in domain_list:
    domain_list.remove("implicit_files")
```